### PR TITLE
[fuchsia_remote_debug_protocol] remove websocket channel dep

### PR DESCRIPTION
**Pull request by jonahwilliams**
Remove the usage of package:web_socket_channel dependency. WIP to allow more important pubspec dependencies to be added first.

### CODE REVIEW

File: packages/fuchsia_remote_debug_protocol/pubspec.yaml [hunk 1]
```diff
@@ -10,70 +10,14 @@ environment:
 dependencies:
   json_rpc_2: 2.2.2
   process: 4.0.0-nullsafety.4
-  web_socket_channel: 1.1.0
```
Comment (dnfield) [inline packages/fuchsia_remote_debug_protocol/pubspec.yaml:13]: I think you need to explicitly add stream_channel now.

**Comment by jonahwilliams** [comment]
This didn't remove as much as I had hoped.

File: packages/fuchsia_remote_debug_protocol/pubspec.yaml (hunk 1 above)
Comment (jonahwilliams) [reply packages/fuchsia_remote_debug_protocol/pubspec.yaml:13]: done

**Review by dnfield — APPROVED** [review]
LGTM I'd you think it's still worth landing.

Cvs we avoid stream_channel?

**Comment by jonahwilliams** [comment]
No, json_rpc_2 depends on it

**Comment by jonahwilliams** [comment]
I'd say go for yours first, this is more minor

**Comment by dnfield** [comment]
Can we drop json_rpc_2?

**Comment by jonahwilliams** [comment]
at some point we should, but thats not a quick fix

**Pull request closed without merging**